app/controllers/v0/sessions_controller.rb

```diff
@@ -48,7 +48,7 @@ def user_attributes
         zip:          attributes['zip']&.first,
         email:        attributes['email']&.first,
         ssn:          attributes['social']&.first,
-        birth_date:   attributes['birth_date']&.first,
+        birth_date:   Time.parse(attributes['birth_date']&.first).utc,
```

Comment:
It's not ID.me that provides it as an array - it's the way the [ruby-saml](https://github.com/onelogin/ruby-saml) parses then spits out the SAML attributes.  There's a `single_value_compatibility` setting that that we can use instead that might clean things up.  OTOH SAML attributes are allowed to come as an array, for example:

`<saml:Attribute name="myArray">
  <saml:AttributeValue>val_1</saml:AttributeValue>
  <saml:AttributeValue>val_2</saml:AttributeValue>
  <saml:AttributeValue>val_2</saml:AttributeValue>
</saml:Attribute>`